docarray/array/mixins/evaluation.py

```diff
@@ -438,12 +456,20 @@ def fuse_matches(global_matches: DocumentArray, local_matches: DocumentArray):
                     new_matches.append(m)
                 query_data[doc.id, 'matches'] = new_matches
 
+        if ground_truth and label_tag in ground_truth[0].tags:
+            max_rel_per_label = dict(Counter([d.tags[label_tag] for d in ground_truth]))
+        elif not ground_truth and label_tag in query_data[0].tags:
+            max_rel_per_label = dict(Counter([d.tags[label_tag] for d in query_data]))
+        else:
+            max_rel_per_label = None
```

Comment:
i don't understand, `max_rel_per_label` is a variable you passed into the function, then what is this `max_rel_per_label`?

Reply:
okay i see, these are two functions

Reply:
can you elaberate a bit the naming, `max_rel_per_label`? why not `num_relevant_documents_per_label` or something like that?

Reply:
ok done